Lib/fontTools/cffLib/__init__.py

```diff
@@ -18,47 +18,66 @@
 # mute cffLib debug messages when running ttx in verbose mode
 DEBUG = logging.DEBUG - 1
 log = logging.getLogger(__name__)
-isCFF2 = False
 
 cffHeaderFormat = """
 	major:   B
 	minor:   B
 	hdrSize: B
 """
 
-defaultMaxStack = 193
-maxStackLimit = 513
+maxStackLimit = 513 
+# maxstack operator has been deprecated. max stack is now always 513.
+
+class CFFContext(object):
```

Comment:
I chose to pass cffCtx with the one field to allow further refactoring. I am considering passing several other fields that get widely used - topDict,  the current privateDict, and adding a getNumRegions that would call the same function on the current private dict. 

The reason to have cffCtx set in the constructors is that some actions, like snapshotting or using a pen, may behave differently if you want the data to act like CFF or like CFF2.